introduction.md

```diff
@@ -0,0 +1 @@
+Hello Deadly Vipers Dojos! This is Hariharan Iyer, seeking to join your elite clan!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.